phoenixRest/models/core/agenda_entry.py

```diff
@@ -71,6 +73,7 @@ def __json__(self, request):
             'description': self.description,
             'location': self.location,
             'time': int(self.time.timestamp()) if self.time is not None else None,
+            'duration': int(self.duration),
```

Comment:
Why do you need to cast it to int? The db type is already `Integer`...

Reply:
I dont know why I cast it to int, maybe because I thought since its an Integer and not a string, why not just use `int(self.duration)`, but I see that it has no effect, so I removed it.